modules/carto/src/index.ts

```diff
@@ -22,19 +22,30 @@ export {
   VectorTileLayer
 };
 
-export {default as BASEMAP} from './basemap';
+export {
+  default as BASEMAP,
+  GOOGLE_BASEMAPS as _GOOGLE_BASEMAPS,
+  getStyleUrl as _getStyleUrl,
+  fetchStyle as _fetchStyle,
+  applyLayerGroupFilters as _applyLayerGroupFilters,
+  STYLE_LAYER_GROUPS as _STYLE_LAYER_GROUPS
+} from './basemap';
 export {default as colorBins} from './style/color-bins-style';
 export {default as colorCategories} from './style/color-categories-style';
 export {default as colorContinuous} from './style/color-continuous-style';
 export {CartoAPIError, fetchMap, query} from './api/index';
+export {fetchBasemapProps} from './api/basemap';
 export type {
   APIErrorContext,
   FetchMapOptions,
   Format,
   MapType,
   RequestType,
   QueryParameters,
-  QueryOptions
+  QueryOptions,
+  Basemap,
+  MaplibreBasemap,
+  GoogleBasemap
```

Comment:
Optional: DeckGL will probably add some MapLibre-specific exports fairly soon, separately from the Mapbox exports. I'm not sure yet whether those will be named "Maplibre" or "MapLibre", and it would be weird to have it be different than DeckGL or to need to change the name in a 9.x release. Perhaps if we don't need to export the "Basemap" types quite yet, we should wait to avoid having to change them later?

Reply:
Good point, i'll export them with _ as experimental.

Maplibre is actually `MapLibre` as it stands everywhere in their webpage, docs and official documentation. Changed to this form everywhere, where we used camelcase